src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -1,15 +1,35 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Bulldozer
- * Do not remove no-args constructor
- */
 public class Bulldozer extends Machine {
+    private int enginePower;
+    private String dumpType;
```

Comment:
Don't forget about access modifiers for your custom fields. It's better to make `enginePower` and `dumpType` private to encapsulate the internal state of the Bulldozer class.